paddlenlp/transformers/xlnet/tokenizer.py

```diff
@@ -334,14 +344,14 @@ def build_offset_mapping_with_special_tokens(self,
         - pair of sequences:    ``A (0,0) B (0,0) (0,0)``
```

Comment:
Are the last tuples the returned ones in the following examples? (the examples looks confusing to me)
"single sequence: X (0,0) (0,0)
pair of sequences: A (0,0) B (0,0) (0,0)"